.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
+.yamllint
+inventory.yml
+playbook.yml
```

Comment:
Wonder if we need to explicitly exclude these files... we're expecting this role to be installed from galaxy, so it's unlikely that these files will exist in the project root

Reply:
Yes, I did that in another commit. Thanks!